raiden/routing.py

```diff
@@ -22,6 +22,7 @@ def get_best_routes(
         amount: int,
         previous_address: typing.Optional[typing.Address],
         config: Dict[str, Any],
+        privkey: bytes = None,
```

Comment:
This is a required argument as far as i can see

Reply:
It is not required if we just use internal routing instead of a pfs. In particular there are a lot of tests that want to call this function and only use internal routing, so I made the argument optional. I changed the function now so that pfs is skipped whenever `privkey` is `None`.

Reply:
I see... then it should be `Optional[bytes]` similar to the `previous_address` argument above it.

Reply:
Done.